crates/bevy_pbr/src/meshlet/from_mesh.rs

```diff
@@ -283,19 +273,14 @@ fn simplify_meshlet_groups(
         }
     }
 
-    // Allow more deformation for high LOD levels (1% at LOD 1, 10% at LOD 20+)
-    let t = (lod_level - 1) as f32 / 19.0;
-    let target_error_relative = 0.1 * t + 0.01 * (1.0 - t);
-    let target_error = target_error_relative * mesh_scale;
-
     // Simplify the group to ~50% triangle count
     // TODO: Simplify using vertex attributes
     let mut error = 0.0;
     let simplified_group_indices = simplify(
         &group_indices,
         vertices,
         group_indices.len() / 2,
-        target_error,
+        f32::MAX,
```

Comment:
After this change, is there a scenario where both of the following happen:

1. simplification fails to reach target triangle count,
2. there are still leftover vertices that are not shared with other meshlets?

For Nanite simplification, all vertices mentioned in point 2 could probably be removed. While this might introduce artefacts, the error metric should show more detailed meshlets in such cases.